frontend/test/resolution/testScopeResolve.cpp

```diff
@@ -1211,6 +1211,58 @@ static void test31() {
   assert(reX.toId() == fn->toFunction()->formal(0)->id());
 }
 
+static void test32a() {
+  printf("test32\n");
+  Context ctx;
+  Context* context = &ctx;
+  ErrorGuard guard(context);
+
+  auto path = UniqueString::get(context, "input.chpl");
+  std::string contents = R""""(
+      module M {
+        proc int.method() {}
+      }
+
+      module O {
+        use M only method;
+      }
+   )"""";
+  setFileText(context, path, contents);
+
+  const ModuleVec& vec = parseToplevel(context, path);
+  assert(vec.size() == 2);
+  const Module* moduleO = vec[1];
+  auto moduleResolutionResults = scopeResolveModule(context, moduleO->id());
+
+  assert(guard.realizeErrors() == 0);
```

Comment:
The destructor should crash the program if there were errors, if you want to save a couple lines.

Reply:
Oh, I didn't know it also calls realize errors. That's why I didn't just use the destructor.